scripts/helper_numpy.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# coding: utf8
+'''Compute a Numpy friendly data-structures from VGGish files
+Example
+-------
+$ cd {repo_root}
+$ ./scripts/helper_numpy.py --csv_file /path/to/sparse-labels.csv
+--vggish_path /path/to/vggish/ --output_file /path/to/output.npz
+
+Any valid output file '*.npz' has to be speficied by the user
+'''
+
+import argparse
+import numpy as np
+import os
+import pandas as pd
+import sys
+import json
+
+
+def main(csvfile, vggishpath, outfile):
+
+    success = []
+    df = pd.read_csv(csvfile)
+    instruments = np.unique(df['instrument'])
+    sample_key = np.unique(df['sample_key'])
+
+    print('Extracting the vggish features...')
+    X = np.empty([20000, 10, 128], dtype=int)
```

Comment:
we could infer the number of rows from number of sample keys?

Reply:
Done.